src/main/java/mate/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final String DRIVER_ID = "driverId";
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long currentDriverID = (Long) req.getSession().getAttribute(DRIVER_ID);
```

Comment:
DRIVER_ID used only once, I think constant is not appropriate here.

Reply:
driverId should be constant according to checklist: Make driverId a constant field in all classes where you use it.